mailbox/store/src/main/java/org/apache/james/mailbox/store/mail/MailboxMapper.java

```diff
@@ -59,8 +61,21 @@
     /**
      * Return the {@link Mailbox} for the given name
      */
-    Mailbox findMailboxByPath(MailboxPath mailboxName)
-            throws MailboxException, MailboxNotFoundException;
+    Mono<Mailbox> findMailboxByPath(MailboxPath mailboxName);
```

Comment:
`Publisher<Mailbox>` ?

Reply:
No IMO this is withing the mailbox module thus we can keep the Mono here as this is an internal API

(internal within mailboxManager)